modules/genomic_browser/php/NDB_Menu_Filter_genomic_browser.class.inc

```diff
@@ -0,0 +1,381 @@
+<?php
```

Comment:
This file seems to have PHPCS run on it, so it should be added to Travis. Something like:

"- vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php modules/genomic_browser"

(to automatically add all PHP from the module.)